R/upload_presentation.R

```diff
@@ -2,14 +2,14 @@
 #'@param file a file path to the presentation for upload
 #'@param ... additional parameters passed to \code{\link[sbtools]{session_check_reauth}}
 #'@importFrom sbtools item_upload_create session_check_reauth current_session
+#'@importFrom mda.streams locate_folder
 #'@export
 upload_presentation <- function(file, ...){
   
   session_check_reauth(...)
-  session = current_session()
   
-  pres_folder <- mda.streams:::get_presentation_id(session)
-  id_out <- item_upload_create(parent_id = pres_folder, file, session)
+  pres_folder <- locate_folder('presentations')
+  id_out <- item_upload_create(parent_id = pres_folder, file)
   url_out <- .url_from_id(id_out)
```

Comment:
there's also now mda.streams:::format_item, which can do this in a slightly fancier way. optional; i guess if we used that then we'd want to make it an exported function in mda.streams, etc.